docs/operator-guides/resumability.md

```diff
@@ -0,0 +1,85 @@
+---
+products: all
+---
+
+# Resumability & Resumable Full Refresh
+
+:::info
+
+This page describes a coming-soon feature. Resumable Full Refresh will be available as of Airbyte v1.0.0.
```

Comment:
https://docs.airbyte.com/contributing-to-airbyte/#documentation States we shouldn't pre-announce. Do we have a timeline for the availability of this feature?

Reply:
Maybe, instead, you can link to the section that lists what sources and APIs support resumable full refresh with something a little like "This feature is currently limited to certain sources and APIs. The section on [enabling resumable full refresh](link it) lists those for which this feature is currently supported"

Reply:
RFR is currently available in Cloud for the supported connectors. It will be available in OSS in 2 - 3 weeks.

I like the suggestion to link to a section where the feature is supported. Is it ok if the linked section is part of this document?

Reply:
I didn't complete my thought: Happy to leave this in a PR!

Thought it would be useful to have this out since the feature is available in various product lines.